GSA/digitalgov.gov
# Bug: Refactor footnote to a component

## Summary

Addressing bug that was causing content to extend past content wrapper 

<details>
  <summary>Before and After</summary>
<img width="1374" alt="Screenshot 2024-07-25 at 10 58 52 AM" src="https://github.com/user-attachments/assets/30f072ba-a32a-4397-bae8-5f432cc5b73a">
<img width="1363" alt="Screenshot 2024-07-25 at 10 59 13 AM" src="https://github.com/user-attachments/assets/5c50fd48-9749-4cd0-94a5-62cc3fa96b21">
</details>

## Solution

_**TL;DR:** Updated `footer` element within the file to be a `div` to match up with best practices and applied styles to the list item to help with word wrapping_

1. Address a few other issues with markup and styles 
1a. The footnote was being applied as a footer, this is not best practice since we are already using a footer 
1b. Updated the footer tag to a div and added BEM naming to this new div and added new styles to the `li`to help with the word wrapping

This solution was chosen for a few reasons, the style solution was to help with the wrapping of the link to help address the issue with content extending past the wrapper and the update to the footer was updated to a div using BEM to align with our projects best practices and also help with mark up and remove a footer that was not truly the footer.






## How To Test

1. visit [preview page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-footnote-update/resources/delivering-digital-first-public-experience/) and adjust screen size to mobile and verify that there is no longer an issue with content going past the wrapper 







Tagging supervisor - @rayestrada
